tests/test_browsers_playlists.py

```diff
@@ -322,10 +357,38 @@ def fns(songs):
             return [song('~filename') for song in songs]
         self.failUnlessEqual(fns(imported.songs), fns(pl.songs))
 
+    @staticmethod
+    def a_delete_event():
+        ev = Gdk.Event()
+        ev.type = Gdk.EventType.KEY_PRESS
+        ev.keyval, accel_mod = Gtk.accelerator_parse("Delete")
+        ev.state = Gtk.accelerator_get_default_mod_mask() & accel_mod
+        return ev
+
     @staticmethod
     def _fake_browser_pack(b):
         app.window.get_child().pack_start(b, True, True, 0)
 
+    class MockConfirmerAccepting(object):
+
+        RESPONSE_INVOKE = Gtk.ResponseType.YES
+
+        def __init__(self, *args):
```

Comment:
This can go, no?

Reply:
I think it might be needed unless the implementation is changed
The Confirmer is passed values needed for the gtk dialog in the function confirm_remove_playlist_dialog_invoke